.github/workflows/optimize-unit-tests.yml

```diff
@@ -49,6 +49,16 @@ jobs:
         retention-days: 7
         if-no-files-found: ignore
 
+    - name: Observe build status
+      if: always()
+      continue-on-error: true
+      uses: ./.github/actions/observe-build-status
+      with:
+        build_status: ${{ contains(steps.*.conclusion, 'failure') && 'failure' || 'success' }}
```

Comment:
Please use the code from https://github.com/camunda/camunda/wiki/CI-&-Automation#metrics-collection which uses `build_status: ${{ job.status }}`